go/noise/laplace_noise_test.go

```diff
@@ -172,6 +172,525 @@ func TestDeltaForThresholdLaplace(t *testing.T) {
 		}
 	}
 }
+func TestInverseCDFLaplace(t *testing.T) {
+	for _, tc := range []struct {
+		desc                     string
+		mean, lambda, prob, want float64
+	}{
+		// 2 high-precision calculated random tests.
+		{
+			desc:   "High-precision random test",
+			mean:   64,
+			lambda: 4,
+			prob:   0.7875404240919761168041,
+			want:   67.4234254367},
+		{
+			desc:   "High-precision random test",
+			mean:   23,
+			lambda: 2,
+			prob:   0.1479685611330654517049,
+			want:   20.564783456,
+		},
+		// Edge cases where p = 0 or p = 1.(Output should be infinite)
+		{
+			desc:   "Negative infinity output when probablity is 0",
+			mean:   0,
+			lambda: 1,
+			prob:   0,
+			want:   math.Inf(-1),
+		},
+		{
+			desc:   "Positive infinity output when probablity is 1",
+			mean:   1,
+			lambda: 1,
+			prob:   1,
+			want:   math.Inf(1),
+		},
+		// Logical testing (with p = 0.5, return value should be mean regardless of lambda)
+		{
+			desc:   "%50 confidence level, different lambda",
```

Comment:
prob is not a confidence level, in particular, prob = 0.5 does not imply 50% confidence.

also it is not clear what "different lambda" is supposed to mean. different from what?

Reply:
Regardless of the parameter Lambda, you should be getting the mean always with a %50 confidence level. That is what is meant by the test. It should be more clearer, I will adjust it.